drivers/wifi/infineon/Kconfig.cyw43xxx

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) 2022 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig WIFI_CYW43XXX
+	bool "Infineon CYW43xxx SoC Wi-Fi support"
+	select THREAD_CUSTOM_DATA
+	select USE_INFINEON_SDIO
+	select WIFI_OFFLOAD
+	select NET_L2_WIFI_MGMT
+	select WIFI_USE_NATIVE_NETWORKING
+	select INFINEON_ABSTRACTION_RTOS
+	depends on DT_HAS_INFINEON_CYW43XXX_WIFI_SDIO_ENABLED
+	help
+	  Enable Infineon CYW43xxx SoC Wi-Fi support.
+
+if WIFI_CYW43XXX
+
+config CYW43XXX_WIFI_SSID
+	string "SSID of Wi-Fi network"
+	help
+	  SSID (network name) for the application to connect to.
+
+config CYW43XXX_WIFI_PASSWORD
+	string "Password (WPA or WPA2) of Wi-Fi network"
+	help
+	  WiFi password (WPA or WPA2) for the example to use.
```

Comment:
Those are not used, so probably can be removed.

Reply:
removed.